keyboards/akko/5075/5075.c

```diff
@@ -0,0 +1,263 @@
+    {0, D_4,   E_4,   F_4},
+    {0, D_5,   E_5,   F_5},
+    {0, D_6,   E_6,   F_6},
+    {0, D_7,   E_7,   F_7},
+    {0, D_8,   E_8,   F_8},
+    {0, D_9,   E_9,   F_9},
+    {0, D_10,  E_10,  F_10},
+    {0, D_11,  E_11,  F_11},
+    {0, D_12,  E_12,  F_12},
+    {0, D_13,  E_13,  F_13},
+    {0, D_14,  E_14,  F_14},
+    {1, D_2,   E_2,   F_2},
+
+    {0, G_1,   H_1,   I_1},
+    {0, G_2,   H_2,   I_2},
+    {0, G_3,   H_3,   I_3},
+    {0, G_4,   H_4,   I_4},
+    {0, G_5,   H_5,   I_5},
+    {0, G_6,   H_6,   I_6},
+    {0, G_7,   H_7,   I_7},
+    {0, G_8,   H_8,   I_8},
+    {0, G_9,   H_9,   I_9},
+    {0, G_10,  H_10,  I_10},
+    {0, G_11,  H_11,  I_11},
+    {0, G_12,  H_12,  I_12},
+    {0, A_15,  B_15,  C_15},
+    {0, G_13,  H_13,  I_13},
+    {1, D_3,   E_3,   F_3},
+
+    {0, J_1,   K_1,   L_1},
+    {0, A_16,  B_16,  C_16},
+    {0, J_2,   K_2,   L_2},
+    {0, J_3,   K_3,   L_3},
+    {0, J_4,   K_4,   L_4},
+    {0, J_5,   K_5,   L_5},
+    {0, J_6,   K_6,   L_6},
+    {0, J_7,   K_7,   L_7},
+    {0, J_8,   K_8,   L_8},
+    {0, J_9,   K_9,   L_9},
+    {0, J_10,  K_10,  L_10},
+    {0, J_11,  K_11,  L_11},
+    {0, J_12,  K_12,  L_12},
+    {1, D_7,   E_7,   F_7},
+    {1, D_4,   E_4,   F_4},
+
+    {0, J_13,  K_13,  L_13},
+    {0, J_14,  K_14,  L_14},
+    {0, J_15,  K_15,  L_15},
+    {0, J_16,  K_16,  L_16},
+
+    {0, G_14,  H_14,  I_14},
+    {0, G_15,  H_15,  I_15},
+    {0, G_16,  H_16,  I_16},
+
+    {0, D_15,  E_15,  F_15},
+    {1, D_6,   E_6,   F_6},
+    {1, D_5,   E_5,   F_5},
+
+    {1, G_1,   H_1,   I_1},
+    {1, G_2,   H_2,   I_2},
+    {1, G_3,   H_3,   I_3},
+    {1, G_4,   H_4,   I_4},
+    {1, G_5,   H_5,   I_5},
+    {1, G_6,   H_6,   I_6},
+    {1, G_7,   H_7,   I_7},
+    {1, G_8,   H_8,   I_8},
+    {1, G_9,   H_9,   I_9},
+    {1, G_10,  H_10,  I_10},
+    {1, G_11,  H_11,  I_11},
+
+    {1, J_1,   K_1,   L_1},
+    {1, J_2,   K_2,   L_2},
+    {1, J_3,   K_3,   L_3},
+    {1, J_4,   K_4,   L_4},
+    {1, J_5,   K_5,   L_5},
+    {1, J_6,   K_6,   L_6},
+    {1, J_7,   K_7,   L_7},
+    {1, J_8,   K_8,   L_8},
+    {1, J_9,   K_9,   L_9},
+    {1, J_10,  K_10,  L_10},
+    {1, J_11,  K_11,  L_11},
+};
+
+#endif
+
+enum __layers {
+    WIN_B,
+    WIN_W,
+    WIN_FN,
+    MAC_B,
+    MAC_W,
+    MAC_FN
+};
+
+// clang-format on
+void led_init_ports(void) {
+    // set our LED pings as output
+    setPinOutput(LED_CAPS_LOCK_PIN); // LED1 Num Lock
+    writePinLow(LED_CAPS_LOCK_PIN);
+    setPinOutput(LED_MAC_OS_PIN); // LDE2 MAC\WIN
+    writePinLow(LED_MAC_OS_PIN);
+    setPinOutput(LED_WIN_LOCK_PIN); // LED3 Win Lock
+    writePinLow(LED_WIN_LOCK_PIN);
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if (res) {
+        writePin(LED_CAPS_LOCK_PIN, led_state.caps_lock);
+        writePin(LED_MAC_OS_PIN, default_layer_state & ((1<<MAC_B)|(1<<MAC_W)));
+        writePin(LED_WIN_LOCK_PIN, keymap_config.no_gui);
+    }
+    return res;
+}
```

Comment:
the caps lock is set by core code.  And the other two options shouldn't be here, as this function only updates when the lock indicator status changes.   These should be moved to a different function (such as `housekeeping_task_kb(void)`).
```suggestion
```